src/configure-webpack/index.js

```diff
@@ -106,7 +99,7 @@ const buildSharedWebpackConfig = (saguiConfig) => {
       // Should first try to resolve loaders nested within Sagui.
       // This fixes an issue in NPM v2 where webpack incorrectly
       // thinks that the package `eslint` is the `eslint-loader`
-      modulesDirectories: [
```

Comment:
Good point, maybe we can update it. I think it is still a safe bet to be explicit about where to look for loaders.